pubspec.yaml

```diff
@@ -8,12 +8,12 @@ environment:
   sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  charcode: ^1.2.0
   collection: ^1.15.0
   source_span: ^1.8.0
   string_scanner: ^1.1.0
   typed_data: ^1.3.0
 
 dev_dependencies:
+  charcode: ^1.2.0
```

Comment:
is this still used in tests or something?

Reply:
Yes, `chunked_coding_test.dart` uses it.